### ISSUE TITLE
Suggest File::Stat method that actually exists

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#15125]
### Before submitting

- [x] I have searched the existing issues and this hasn't been reported yet.
- [x] I am running the latest stable version of RuboCop.

### Issue type

Incorrect autocorrect

### Cop name

Style/ZeroLengthPredicate

### Steps to reproduce the problem

```ruby
#!/bin/env ruby

stat = File.stat('.')
stat.size.zero?
stat.empty?
```

### Expected behavior

```
Use `!size?` instead of `size.zero?`
```

### Actual behavior

```
Use `empty?` instead of `size.zero?`
```

```
undefined method 'empty?' for an instance of File::Stat (NoMethodError)
```

### RuboCop configuration

```yaml

```

### RuboCop version

```shell
System Version: macOS 26.3.1 (a) (25D771280a)
Model Identifier: MacBookAir10,1
ruby 4.0.3 (2026-04-21 revision 85ddef263a) +PRISM [arm64-darwin25]
rubocop     2026-04-24 1923d3f28
```
